clang_format/fix_formatting.sh

```diff
@@ -54,24 +54,35 @@ while test $# -gt 0; do
             exit 0
             ;;
         -a|--all)
-            echo "Formatting all files..."
-
-            # Generate extension string
-            # Formatted as -iname *.EXTENSION -o
-            EXTENSION_STRING=""
-            for value in "${EXTENSIONS[@]}"
-            do
-                EXTENSION_STRING="$EXTENSION_STRING -iname *.$value -o"
-            done
-            
-            # Find all the files that we want to format, and pass them to
-            # clang-format as arguments
-            # We remove the last -o flag from the extension string
-            find $CURR_DIR/../src/ ${EXTENSION_STRING::-2} \
-                | xargs $CURR_DIR/clang-format-$CLANG_VERSION -i -style=file
-
-            shift
-            ;;
+            echo "WARNING: "-a" will format all the files and can be very slow! Please consider using "-b" instead!"
+
+            echo "Are you sure you want to format all files (y/n)?"
+            read continueFormat
+            if [ "$continueFormat" == "n" ]; then
+		exit 0
+	    elif [ "$continueFormat" == "y" ]; then
```

Comment:
Indenting

Reply:
Specifically the indenting is inconsistent, and there seem to be tabs mixed in as well. Please indent using 4 spaces.

Reply:
I fixed indenting in the most recent push